addons/mail/static/src/components/discuss_sidebar/discuss_sidebar.xml

```diff
@@ -3,21 +3,21 @@
 
     <t t-name="mail.DiscussSidebar" owl="1">
         <t t-if="discussView">
-            <div name="root" class="o_DiscussSidebar" t-attf-class="{{ className }}" t-ref="root">
+            <div name="root" class="o_DiscussSidebar d-flex flex-column" t-attf-class="{{ className }}" t-ref="root">
                 <div class="d-flex justify-content-center">
                     <button class="o_DiscussSidebar_startAMeetingButton btn btn-primary rounded" title="Start a meeting" t-on-click="discussView.onClickStartAMeetingButton">
                         Start a meeting
                     </button>
                 </div>
-                <hr class="o_DiscussSidebar_separator"/>
-                <div class="o_DiscussSidebar_category o_DiscussSidebar_categoryMailbox">
+                <hr class="o_DiscussSidebar_separator w-100 border-top"/>
+                <div class="o_DiscussSidebar_category o_DiscussSidebar_categoryMailbox d-flex flex-column flex-grow-0">
                     <DiscussSidebarMailbox threadLocalId="messaging.inbox.localId"/>
                     <DiscussSidebarMailbox threadLocalId="messaging.starred.localId"/>
                     <DiscussSidebarMailbox threadLocalId="messaging.history.localId"/>
                 </div>
-                <hr class="o_DiscussSidebar_separator"/>
+                <hr class="o_DiscussSidebar_separator w-100 border-top"/>
                 <t t-if="messaging.models['Thread'].all(thread => thread.isPinned and thread.model === 'mail.channel').length > 19">
-                    <input class="o_DiscussSidebar_quickSearch" t-on-input="discussView.onInputQuickSearch" placeholder="Quick search..." t-ref="quickSearchInput" t-esc="discussView.discuss.sidebarQuickSearchValue"/>
+                    <input class="o_DiscussSidebar_quickSearch rounded-lg border-0 mx-4 mb-2 px-3 py-1" t-on-input="discussView.onInputQuickSearch" placeholder="Quick search..." t-ref="quickSearchInput" t-esc="discussView.discuss.sidebarQuickSearchValue"/>
```

Comment:
Hmm, in this case, adding a `form-control` class would force us to use additional classes (`w-auto h-auto`) plus new lines of css to override the default css of this new class... 🤔 

Let's see what does @odoo/rd-discuss think about this 👍

Reply:
@alexkuhn , I got you point and I do agree that `form-control` suits better "complete forms".

At the same time I don't find feasible to let browsers design these elements; it decrease consistency and really complicates maintenance...  especially considering that soon we'll have to implement dark-mode just using variables, there is no plan for a specific stylesheet to be loaded on top.

Also, the current implementation on master is not very readable and doesn't provide any user feedback on :focus :/

`form-control` was a compromise to stay within the task's scope and solve these issues, my ideal implementation would be a `o_input_[variation]` class, but it's premature... 

cc. @seb-odoo

Reply:
Ok, I don't have better alternatives than either keep current code or use `form-control` in the short term.
The most important thing is being aware of pros/cons with either solution, and be prepared for future improvements.

My comment was to point-out that `form-control` is more specific than how we use, so it's very likely to annoy us. Personally, and supposedly, if I get too many behavioural issues with `form-control`, then I would definitely trade browser style consistency for behavioural correctness.

Reply:
Ok, thanks @alexkuhn .

> if I get too many behavioural issues with form-control, then I would definitely trade browser style consistency for behavioural correctness.

I understand. The end-goal is to have fields designed consistently across  the entire UI, but at the moment there is too much going on to implement  a `o_input_[variation]` class (SCSS Ref, migration to BS5, UI redesign... ).

`form-control` wold be a good compromise to solve minors on master, improve consistency (eg. it gets the right `:focus` color automatically) and provides potential support for dark-mode thanks to variables.
There may be caveats, but I think it's worth a try.

@karlosvalverde , could you do a test?
thanks

Reply:
> Ok, thanks @alexkuhn .
> 
> > if I get too many behavioural issues with form-control, then I would definitely trade browser style consistency for behavioural correctness.
> 
> I understand. The end-goal is to have fields designed consistently across the entire UI, but at the moment there is too much going on to implement a `o_input_[variation]` class (SCSS Ref, migration to BS5, UI redesign... ).
> 
> `form-control` wold be a good compromise to solve minors on master, improve consistency (eg. it gets the right `:focus` color automatically) and provides potential support for dark-mode thanks to variables. There may be caveats, but I think it's worth a try.
> 
> @karlosvalverde , could you do a test? thanks

I've pushed a temp commit with some changes for you to have a look.

As you'll see, `form-control` replaces the outline issues on `:focus`, which justifies the css line deletion of the conversation below, but at the same time it adds a problem, which is a change on the default colour of the placeholder text in the input.

In this case, if we want to keep the design of the input as similar as possible to its current version, we should add css lines to set properly the placeholder colour. Also, since we are using a `border-0`, the dynamic colour change on `input :focus` won't be noticeable. 

Consequently, we could change the final design, removing the `border-0` class (in order to have this component with a visible border, as we have in other related components, such as `o_DiscussSidebarCategory_addingItemInput`), and keeping the placeholder text darker, as `form-control` does by default.

What do you think? 

@odoo/rd-discuss

Reply:
@karlosvalverde , ok for me.
you could keep just `o_DiscussSidebar_quickSearch form-control mb-2 mx-4 rounded-lg w-auto` classes IMO